composer.json

```diff
@@ -95,7 +95,8 @@
         "vatsim/xml": "dev-main",
         "watson/rememberable": "^5.0",
         "wikimedia/composer-merge-plugin": "^1.4",
-        "wohali/oauth2-discord-new": "^1.0"
+        "wohali/oauth2-discord-new": "^1.0",
+        "restcord/restcord": "dev-develop"
```

Comment:
Is there something on the `develop` branch that is required for this to work? Would be better to fix for a version if possible?

Reply:
The restcord devs are being a bit silly and refusing to merge Laravel 8 support into the main branch, I'll see if that's changed

Reply:
An alternative is https://github.com/bumbummen99/restcord, which someone made with the changes needed and isn't being touched.

Reply:
@cocoiadrop Could we just hit the Discord API ourselves with the Http client to grab this? Ref: https://discord.com/developers/docs/resources/user#get-user?

Reply:
@CalumTowers That's a far better idea actually -- done! https://github.com/VATSIM-UK/core/pull/2029/commits/73307534eb7dbd19b819edadb22bbaacbf48c767

Reply:
@cocoiadrop Much nicer, thanks!